organization/_posts/2024/2024-02-29-coordination.md

```diff
@@ -0,0 +1,230 @@
+>  
+> In the meantime, we have analysed the workshop and decided to set up a number of working groups to prepare a White Paper on European Federated Computing in 2024 as input to the next JENA meeting in early 2025.
+>  
+> We cordially invite you to participate in the working groups and support us in this process.
+> After an appropriate registration period, those registered will be contacted by the chairs for a first meeting of the working group.
+>  
+> Registration is possible via indico (see below). There you also will find a preliminary description of the topic of the working group and a summary of the Bologna workshop, which you can also see here.
+> HPC                   <https://indico.scc.kit.edu/e/JENA_computing_wp1/>
+> 
+> Software              <https://indico.scc.kit.edu/e/JENA_computing_wp2/> --> mailing list exists, ask Graeme 
+> 
+> Data                  <https://indico.scc.kit.edu/e/JENA_computing_wp3/>
+> 
+> ML                     <https://indico.scc.kit.edu/e/JENA_computing_wp4/>
+> 
+> TDE                    <https://indico.scc.kit.edu/e/JENA_computing_wp5/>
+> 
+>  
+> Please register for one or more of these working groups via Indico as soon as possible.
+> As we will be distributing this invitation widely across all three JENA communities, you may receive it more than once - we apologise for this.
+> However, please feel free to forward this invitation to specific colleagues that you think would be interested and helpful to our mission.
+>  
+> With our best regards
+> Andreas Haungs for the JENA Computing Working group
+
+Meetings appear on this indico category: <https://indico.cern.ch/category/18013/>, if you aren't sure of how to join WGs other than WG2, ask Caterina and Graeme and they can find out (not sure the KIT indicos have by now decoupled from the mailing lists that are being created to distribute information). 
+
+### Google Summer of Code 2024
+
+HSF accepted as organization. 
+Number of slots only known after student rankings.
+
+Some communication (from the candidates) is lost in CERN email security-related issues / filters. Maybe a solution could be to use Google / GitHub classroom to recruit and select candidates. 
+
+Q&A from later on in the meeting:
+
+  - all-remote program
+  - Comment from JimP: the all-remote format works well, the students are ready for that kind of interaction, and we do have some continuity: usually GSoC the first year, then re-hire them as IRIS-HEP Fellows, then other mechanisms, up to 3 years or so
+  - Caterina/Benedikt: you get very good computer scientists that can solve standalone problems. 
+  - Saptaparna: connection to experiments? A: does not need to be too tight. 
+  - Not too late for students to join project proposals, broad advertising. 
+
+### HSF Evolution
+
+Work in progress: next meeting of the core coordination team next week, producing some text for everyone to look at. 
+
+## Working Group Updates
+
+### General Standing Reminders
```

Comment:
I always take these general standing reminders out of the published minutes - there's no sense in having them on the website every time